source/draft_users.rst

```diff
@@ -0,0 +1,179 @@
+password      string        true       Password for user login         Stored as bcrypt hash; not returned on get
+display_name  string        false      Name used for public display
+active        boolean       false      Whether user is can log in      Used by admins to invalidate users
+admin         boolean       false      Whether user has admin rights   Can only be set by another admin
+email         string        false      Email address of user
+created_at    ISO date      false      The date the user was created   Automatically set by the server, unchangeable
+deleted_at    ISO date      false      The date the user was deleted   Automatically set by the server upon delete
+============  ========  =============  ==============================  =============================================
+
+It is worth noting that, because users are updated "in-place" (i.e. without an audit trail
+or revision system), there are no ``uuid``, ``revision``, or ``updated_at`` fields.
+
+-----------
+
+Admin Users
+-----------
+
+An admin user is defined by the boolean ``admin`` field. Admin users are able to access
+any endpoint, and manage all users (including being the only users who can add users or
+promote them to admin).
+
+---------------
+
+/users Endpoint
+---------------
+
+User management involves a new endpoint, at ``/users``. In most respects it is similar to
+the other endpoints, with certain key differences.
+
+GET /users
+~~~~~~~~~~
+
+Returns a list of all users in the database:
+
+.. code-block:: javascript
+
+    [
+      {
+        "username": "example",
+        "displayname": "X. Ample User",
+        "email": "example@example.com",
+        "active": true,
+        "admin": false,
+        "created_at": "2015-02-29",
+        "deleted_at": null
+      },
+      {
+        ...
+      },
+      ...
+    ]
+
+This endpoint can be accessed by any authenticated user.
+
+GET /users/:username
+~~~~~~~~~~~~~~~~~~~~
+
+Returns a specific user from the database.
+
+.. code-block:: javascript
+
+    {
+      "username": "example",
+      "displayname": "X. Ample User",
+      "email": "example@example.com",
+      "active": true,
+      "admin": false,
+      "created_at": "2015-02-29",
+      "deleted_at": null
+    }
+
+This endpoint can be accessed by any authenticated user, and may be used to look up any
+user.
+
+POST /users
+~~~~~~~~~~~
+
+Create a new user.
+
+Request:
+
+.. code-block:: javascript
+
+    {
+      "username": "example",
+      "password": "password",
+      "displayname": "X. Ample User",
+      "email": "example@example.com"
+    }
+
+Response:
+
+.. code-block:: javascript
+
+    {
+      "username": "example",
+      "displayname": "X. Ample User",
+      "email": "example@example.com",
+      "active": true,
+      "admin": false,
+      "created_at": "2015-02-29",
+      "deleted_at": null
+    }
+
+This endpoint may only be accessed by admins. It is therefore recommended that admins
```

Comment:
It's not really the endpoint that can only be accessed by admins, but the create functionality